time detectedTime
float32 x
float32 y
float32 z
float32 width
float32 height
float32 orientation

#0-not yet on belt, 1- on belt, 2 - removed from belt, 3 - undefined
int8 state

#0-PET,1-HDPE,2-others
int8 plastictype

